modules/nw-networkpolicy-about.adoc

```diff
@@ -1,6 +1,7 @@
 // Module included in the following assemblies:
 //
-// networking/configuring-networkpolicy.adoc
+// * networking/configuring-networkpolicy.adoc
+// * post_installation_configuration/network-configuration.adoc
```

Comment:
This sentence needs update: "The Kubernetes `v1` NetworkPolicy features are available in {product-title}
except for egress policy types and IPBlock."

IPBlock is supported in NetworkPolicy with limitations for OpenshiftSDN; it supports IPBlock without except clauses. If you create a policy with an ipBlock section including an except clause, the SDN Pods log warnings, and the entire ipBlock section of that policy is ignored.